### PR TITLE
Description about how to access configuration variable values

### PR DESCRIPTION
**Pull request by Aminolroaya**
Might be helpful for Sherpa newbies.

### CODE REVIEW

File: docs/How_To/Slack Bot/1_slackbot_workspace.rst [hunk 1]
```diff
@@ -59,8 +59,21 @@ After you clone the repository, you can find the slackbot project under `apps/sl
...
-   into your own ``slackbot/.env`` file, and then modify your configuration as needed.
-   Remember not to share your private keys with anyone else.
+   into your own ``slackbot/.env`` file.
+    :: 
+
+        cp .env-sample .env
+   * Then, modify your configuration as needed. You can use the ``OS Module`` to set the configuration variables as below. Remember not to share your private keys with anyone else.
+    :: 
+
+           import os
```
Comment (20001LastOrder) [inline docs/How_To/Slack Bot/1_slackbot_workspace.rst:69]: I think this part is already done in the `config.py` file by the `load_dotenv` function. Weren't you able to get the environment variables setup without manually setting them with the os module?

Comment (Aminolroaya) [reply docs/How_To/Slack Bot/1_slackbot_workspace.rst:69]: Thanks. I assume the user first needs to set environment variables? What if they did not? We can also note to set env variables before running the app?

Comment (20001LastOrder) [reply docs/How_To/Slack Bot/1_slackbot_workspace.rst:69]: Yes, the user needs to create a file called `.env` and add the environment variables there following the format in `.env_sample`. Maybe we can make this more explicit

**Review by 20001LastOrder — APPROVED** [review]
LGTM. Thanks for writing the clarification.

**Pull request merged**